src/command_modules/azure-cli-cognitiveservices/azure/cli/command_modules/cognitiveservices/_help.py

```diff
@@ -44,6 +44,10 @@
 helps['cognitiveservices account create'] = """
     type: command
     short-summary: Create an Azure Cognitive Services account.
+    parameters: 
+        - name: --kind
+          populator-commands: 
+            - az cognitiveservices account list-kinds
```

Comment:
Similarly, create and update should have the populator command for `list-skus`

Reply:
okay, but right now `list-skus` can only be added for `update`, because it require an existing account name.
(I believe `list-skus` should be able to be improved by using similar implementations with `list-kinds`, but because it will use a totally different API, so need to confirm with PM and team.)

Reply:
How do you supply it for create then?

Reply:
They are required parameters on create. I'm not sure what you mean when you say you don't supply them.  Are you saying people just have to know because there's no way to query them?

Reply:
@tjprescott yes that's what I mean. They have to somehow know it. That's the reason we need to improve list-skus to make it not rely on an existing account.

Reply:
Gotcha. Thanks.